allure-report-face/src/components/widget-overview/OverviewWidget.js

```diff
@@ -1,7 +1,53 @@
 import './styles.css';
 import {ItemView} from 'backbone.marionette';
 import template from './OverviewWidget.hbs';
+import c3 from 'c3';
```

Comment:
Tooltip looks inconsistent with our other charts.
![screenshot from 2016-05-20 11-15-58](https://cloud.githubusercontent.com/assets/812240/15423202/b1419f22-1e7c-11e6-8dc7-686317fd7a1d.png)

It is not hard to implement it with raw d3: http://bl.ocks.org/mbostock/3887193
No need to use c3 at all